src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  margin: auto;
+  padding: 10px;
+  max-width: $max-width;
+  display: flex;
+  flex-wrap: wrap;
```

```suggestion
.calendar {
  display: flex;
  flex-wrap: wrap;
  
  max-width: $max-width;
  margin: auto;
  padding: 10px;
  
```
[Here](https://codeguide.academy/html-css.html#css-order) is goo recomendation about code style :simple_smile: